packages/ui/src/components/marquee/marquee.tsx

```diff
@@ -0,0 +1,57 @@
+import { Children, ReactNode } from 'react';
+
+import { classes } from '../../utils';
+
+interface MarqueeProperties {
+  children: ReactNode;
```

Comment:
maybe we can accept:
`items: ReactNode[]` so you don't have to use `Children.map` but normal map